prow/apis/prowjobs/v1/types.go

```diff
@@ -319,7 +322,7 @@ func (r Refs) String() string {
 
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
-// VirtualMachineList is a list of VirtualMachine resources
+// ProwJobList is a list of ProwJob resources
```

Comment:
!? now I want to look at the history for this line :joy:

Reply:
:man_facepalming: